dces-drc-integration/build.gradle

```diff
@@ -127,7 +129,16 @@ dependencies {
 	testImplementation "com.squareup.okhttp3:mockwebserver:$versions.mockwebserverVersion"
 
 	testCompileOnly 'org.projectlombok:lombok'
+	testCompileOnly 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
```

Comment:
It's only used by the rest-assured code, for accessing the fdc item details. 

There's no code handling it in the main, so I kept in the test compliation.

Reply:
OK, but `testCompileOnly` means its on the testing compile-time classpath, but not the testing runtime classpath (https://docs.gradle.org/current/userguide/dependency_configurations.html#sec:what-are-dependency-configurations) - surely, it would be `testImplementation` if the tests needed it.

`testCompileOnly` would be used for libaries like Lombok where it's needed for the annotations and processor at compile time, but those annotations are not needed at runtime (and there's no runtime library).

That's not how Jackson works AFAIK (unless you're just using some compile-time annotations from that library, which aren't needed at runtime?) - in which case they might not be needed at all (unless there's an annotation processor involved).

Reply:
Yep, it's un-needed. So chucked.